pkg/scheduler/framework/arguments.go

```diff
@@ -85,6 +85,26 @@ func (a Arguments) GetBool(ptr *bool, key string) {
 	*ptr = value
 }
 
+// GetString get the bool value from string
```

Comment:
The meaning of the comment does not correspond to the function of the function

Reply:
updated @hwdef